pkg/manager/member/tikv_scaler.go

```diff
@@ -94,6 +94,14 @@ func (tsd *tikvScaler) ScaleIn(meta metav1.Object, oldSet *apps.StatefulSet, new
 
 	switch meta.(type) {
 	case *v1alpha1.TidbCluster:
+		tc ,_ := meta.(*v1alpha1.TidbCluster)
+		storesInfo, err := controller.GetPDClient(tsd.pdControl, tc).GetStores()
+		if err != nil {
+			return fmt.Errorf("failed to get stores info when scaling in TiKV")
+		}
+		if storesInfo.Count <= 3 {
+			return fmt.Errorf("it is not allowed to scale in TiKV when stores number(%d) <= 3", storesInfo.Count)
```

Comment:
No need to return error because it will cause the TC to be re-synced, just print the error and return `nil`.
Please also add info of TC.

Reply:
> No need to return error because it will cause the TC to be re-synced, just print the error and return `nil`.
> Please also add info of TC.

@DanielZhangQD Although the TC will be re-synced no matter whether error returns or not, it will be more logical and more friendly to write unit test ,